src/main/java/com/example/school/model/Mark.java

```diff
@@ -0,0 +1,23 @@
+package com.example.school.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.UUID;
+
+import static java.util.UUID.randomUUID;
+
+@Getter
+@Setter
+@Builder
+@AllArgsConstructor
+public class Mark {
+    @Builder.Default
+    private UUID id = randomUUID();
+    private Lesson lesson;
+    private Student student;
+    private int value;
+    private boolean isPresent;
```

Comment:
it would be better to create different model to track absentee in next PR and remove field here `isPresent`.

Reply:
Or you can rename this model somehow. Because now it seems like this model is responsible for mark only.